### PR TITLE
Add components

### PR DESCRIPTION
**Pull request by devMunyi**
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React class-based component that renders a calculator UI

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,30 @@
...
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+table {
+  width: 50% !important;
+  font-size: 25px;
+  margin: 50px auto;
+  color: white;
```
Comment (GzimAsani) [inline src/App.css:6]: - [ ] All the boilerplate that got created by the CRA (texts, images, and styles) need to be deleted. Example: logos, favicon

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,11 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <Calculator />
     </div>
   );
 }
```
Comment (GzimAsani) [inline src/App.js]: - [ ] The app component should be a class-based component. Kindly update it

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @devMunyi            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling, it matches the design :muscle: 

 ## Required Changes 

### Note this project should be done as one project react/rails.
- [x] Check the comments below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (devMunyi) [reply src/App.css:6]: `Please update your readme file and add the comment that the users can run in order to test your app`
Please clarify what you mean, I don't get it.

Comment (GzimAsani) [reply src/App.css:6]: That was my mistake sorry, it wasn't meant to be there

Comment (devMunyi) [reply src/App.css:6]: Okay, no worries.

**Review by Wes-Isaac — APPROVED** [review]
Hi @devMunyi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter error ✅.
- Use of class-based component 🚀.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by devMunyi** [comment]
Thanks @Wes-Isaac

**Pull request merged**